app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controller");
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/topics', getTopics)
+
+//err handling
+app.all('*', (req, res, next) => {
+  if(!req.status){
```

Comment:
You don't need the if(!req.status) it will only ever get here if the endpoint hasn't been matched